yarn.lock

```diff
@@ -15657,7 +15657,7 @@ react-a11y@^0.3.4:
   dependencies:
     object.assign "^4.0.3"
 
-react-ace@^10.1.0:
+react-ace@10.1.0:
```

Comment:
There are still `ace-builds@1.10.1` and `ace-builds@^1.4.14` in your yarn.lock.

```yaml
react-ace@10.1.0:
  version "10.1.0"
  resolved "https://registry.yarnpkg.com/react-ace/-/react-ace-10.1.0.tgz#..."
  integrity sha512-VkvUjZNhdYTuKOKQ...==
  dependencies:
    ace-builds "^1.4.14"
```

Reply:
thanks